<?xml version="1.0" ?>

<launch>

  <arg name="simulation" default="false"/>

  <group unless="$(arg simulation)">
    <!-- Launch socketcan bridge node -->
    <node pkg="socketcan_bridge" name="socketcan_bridge_node" type="socketcan_bridge_node">
    </node>
    
    <!-- Launch driver for each motor -->
    <node pkg="ros_talon" type="servo_pos" name="motor1" output="screen">  
      <param name="motor_nb" value="1"/>
    </node>
    
    
    <!---
    <node pkg="ros_talon" type="direct_drive" name="motor3" output="screen">
      <param name="motor_nb" value="3" type="str"/>
    </node> 

    <node pkg="ros_talon" type="direct_drive" name="motor4" output="screen">
      <param name="motor_nb" value="4" type="str"/>
    </node> 

    <node pkg="ros_talon" type="direct_drive" name="motor5" output="screen">
      <param name="motor_nb" value="5" type="str"/>
    </node> 

    <node pkg="ros_talon" type="direct_drive" name="motor6" output="screen">
      <param name="motor_nb" value="6" type="str"/>
    </node> -->
  </group>
</launch>